src/Features/ERDDAP/Platform/Observations/Table/table.tsx

```diff
@@ -17,7 +20,7 @@ const timeDelta = 15 * 60 * 1000
  * Recent platform observation values
  * @param platform
  */
-export const ErddapObservationTable: React.SFC<Props & RenderProps> = ({ platform, unitSelector, unit_system }) => {
+export const ErddapObservationTable: React.SFC<Props> = ({ platform, unitSelector, unit_system }) => {
```

Comment:
Codacy found an issue: [Identifier 'unit_system' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)